zcash_client_sqlite/src/wallet/db.rs

```diff
@@ -1160,26 +1162,27 @@ WITH unioned AS (
            sent_notes.memo              AS memo,
            NULL                         AS recipient_key_scope
     FROM sent_notes
-    JOIN transactions
-        ON transactions.id_tx = sent_notes.transaction_id
+    JOIN transactions t
+        ON t.id_tx = sent_notes.transaction_id
     LEFT JOIN v_received_outputs ro ON ro.sent_note_id = sent_notes.id
     -- join on the accounts table to obtain account UUIDs
     LEFT JOIN accounts from_account ON from_account.id = sent_notes.from_account_id
 )
 -- merge duplicate rows while retaining maximum information
 SELECT
     transaction_id,
-    MAX(txid),
-    MAX(mined_height) AS mined_height,
+    MAX(txid)                   AS txid,
+    MAX(mined_height)           AS tx_mined_height,
+    MIN(trust_status)           AS tx_trust_status,
```

Comment:
- Document `tx_trust_status` in the documentation for this view.
- Document the addition of `tx_trust_status` in the changelog.
- Fix the changelog now that the newly-added `mined_height` has been renamed to `tx_mined_height`.